### PR TITLE
[2/n][pipeline-gen] Convert default values into constant vars

### PR DESCRIPTION
**Pull request by khluu**
- Convert all default values used in plugin config into constant vars so it's easier to test

### CODE REVIEW

File: scripts/pipeline_generator/plugin.py [hunk 1]
```diff
@@ -88,6 +95,7 @@ class KubernetesPluginConfig(BaseModel):
 
 
 def get_kubernetes_plugin_config(container_image: str, test_bash_command: List[str], num_gpus: int) -> Dict:
+    test_bash_command[-1] = f'"{test_bash_command[-1]}"'
```
Comment (aslonnie) [inline scripts/pipeline_generator/plugin.py:98]: what is this line doing?

File: scripts/tests/pipeline_generator/test_plugin.py [hunk 2]
```diff
@@ -20,7 +20,7 @@ def test_get_kubernetes_plugin_config():
                 "containers": [
                     {
                         "image": docker_image_path,
-                        "command": [" ".join(test_bash_command)],
+                        "command": ['bash -c "echo A"'],
```
Comment (aslonnie) [inline scripts/tests/pipeline_generator/test_plugin.py:23]: I do not quite understand the change in this file.

**Comment by khluu** [comment]
I just removed the quote thing from this PR

**Review by aslonnie — APPROVED** [review]
could you also update the PR description?

**Pull request merged**